server/src/user/user.service.ts

```diff
@@ -112,4 +112,43 @@ export class UserService {
       throw error;
     }
   }
+
+  async deleteUser(userId: number, houseId: number) {
+    try {
+      const userHouse = await this.prisma.userHouse.findFirst({
+        where: { userId, houseId },
+      });
+
+      if (!userHouse) {
+        throw new NotFoundException(
+          `User house not found for user ID ${userId} and house ID ${houseId}`,
+        );
+      }
+
+      // console.log('use House>>', userHouse);
```

Comment:
Delete it!

Reply:
Fix it.
c7fbf3878fea460517f6ab3c4c2edbbeceb299ca